app/helpers/tags_helper.rb

```diff
@@ -0,0 +1,9 @@
+module TagsHelper
+  def tags_in_pillboxes(tags)
+    sanitize tags.map { |tag| tag_in_pillbox(tag)}.join(' ')
```

Comment:
пробелчик перед закрывающей фигурной
```suggestion
    sanitize(tags.map { |tag| tag_in_pillbox(tag) }.join(' '))
```